### PR TITLE
Logical camera image headers

### PR DESCRIPTION
**Pull request by jaybrecht**
Added a header to both the basic logical camera image and advanced logical camera image messages.

### CODE REVIEW

**Comment by zeidk** [comment]
The time stamp is not updated and is always set to 0. This is true for both the logical and the advanced logical cameras.

```
---
header:
  stamp:
    sec: 0
    nanosec: 0
  frame_id: basic_logical_camera_0_frame
part_poses:
- position:
    x: 1.0800002818976169
    y: 0.5150000713808152
    z: -0.2060066545377186
  orientation:
    x: 8.563799563209906e-06
    y: -0.7071045326060549
    z: 6.442450900670502e-06
    w: 0.7071090296786833
- position:
    x: 1.079999561206743
    y: 0.1550000713791685
    z: -0.20600677923982297
  orientation:
    x: -2.7387384657138466e-06
    y: -0.707112455540799
    z: -4.86002105489512e-06
    w: 0.7071011067647549
- position:
    x: 1.0799972657939225
    y: 0.5150001960856723
    z: -0.5660066545307398
  orientation:
    x: -1.920765885633968e-06
    y: -0.707111625126857
    z: -4.0420512293159855e-06
    w: 0.7071019371988935
- position:
    x: 1.0800077935185652
    y: 0.15500019608856302
    z: -0.5660067793083816
  orientation:
    x: 8.512969868981192e-06
    y: -0.7071045326854007
    z: 6.39162126229756e-06
    w: 0.7071090296004129
tray_poses: []
sensor_pose:
  position:
    x: -2.286
    y: 2.96
    z: 1.8
  orientation:
    x: -0.7071045443232116
    y: -1.2248082623837582e-07
    z: 0.7071090180427863
    w: -1.2246817995071293e-07
---
```

File: ariac_sensors/src/ariac_logical_camera_plugin.cpp [hunk 1]
```diff
@@ -120,6 +123,8 @@ void AriacLogicalCameraPluginPrivate::OnUpdate()
 
   const auto & image = this->sensor_->Image();
 
+  gazebo::common::Time sensor_update_time = this->sensor_->LastMeasurementTime();
```
Comment (zeidk) [inline ariac_sensors/src/ariac_logical_camera_plugin.cpp]: I would suggest you print sensor_update_time to see if the correct value is assigned here. This seems an issue which can be resolved with a static cast.

Comment (jaybrecht) [reply ariac_sensors/src/ariac_logical_camera_plugin.cpp]: The `LastMeasurementTime()` function apparently doesn't work for logical cameras. I have switched to ros time from the node and it is working now.

**Review by zeidk — APPROVED** [review]
Time stamp is now updated for both the logical and advanced cameras.

**Pull request merged**